docs/modules/demos/pages/data-lakehouse-iceberg-trino-spark.adoc

```diff
@@ -99,7 +108,7 @@ this is only supported in Spark. Trino is https://github.com/trinodb/trino/issue
 If you want to read more about the motivation and the working principles of Iceberg, please have a read on their
 https://iceberg.apache.org[website] or https://github.com/apache/iceberg/[GitHub repository].
 
-== Listing Deployed Stacklets
+== List the deployed Stackable services
 
 To list the installed installed Stackable services run the following command:
```

Comment:
suggestion: Maybe use the stacklets term in both the headline and the paragraph.

Reply:
The Stacklets term was removed in favor of Services.